ee/docker-compose.ch.yml

```diff
@@ -32,7 +32,7 @@ services:
             CLICKHOUSE_VERIFY: 'False'
             KAFKA_URL: 'kafka://kafka'
             REDIS_URL: 'redis://redis:6379/'
-            SECRET_KEY: '<randomly generated secret key>'
+            SECRET_KEY: 'alsdfjiosdajfklalsdjkf'
```

Comment:
Indeed, having errors thrown because of `<randomly generated secret key>` is annoying, but the point is that users then know they should update for their own safety. Perhaps moving towards @Twixes  approach from https://github.com/PostHog/posthog/pull/1704 is best, while keeping `<randomly generated secret key>` here for safety.

Reply:
For what it's worth only we will be using this docker-compose file. This is not for general consumption.

Reply:
Hmm yeah that's true... I guess it's not a problem then

Reply:
Ah yep I didn't mean to commit this but it was annoying. If James G is happy I can leave this in

Reply:
Ah yep I didn't mean to commit this but it was annoying. If James G is happy I can leave this in

Reply:
yeah doesn't bother me any